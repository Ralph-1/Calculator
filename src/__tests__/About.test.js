import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Quote from '../components/Quote';

describe('Home component test', () => {
  test('should renders welcome text', () => {
    render(<Quote />);
    const text = screen.getByText('Quote');
    expect(text).toBeInTheDocument();
  });

  test('should render the expected output', () => {
    const component = renderer.create(
      <Quote />,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});