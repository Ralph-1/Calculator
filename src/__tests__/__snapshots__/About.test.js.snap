// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component test should render the expected output 1`] = `
Array [
  <hr />,
  <h2
    className="text-center"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
  </h2>,
  <hr />,
]
`;
